### PR TITLE
Mobile

### PR DESCRIPTION
**Pull request by jennypollack**

### CODE REVIEW

File: docs/guide/deeplinking.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Deeplinking
+
+[Create deeplinks for your application here](https://metamask.github.io/metamask-deeplinks/#)
+
+Deeplinks allows for instant invocation of the user's preferred wallet application (even if it is a webapp or a swarm đapp), with the correct parameterization of the transaction that can then only be confirmed by the (authenticated) user.
+
+URLs embedded in QR-codes, hyperlinks in web-pages, emails or chat messages provide for robust cross-application signaling between very loosely coupled applications.
```
Comment (omnat) [inline docs/guide/deeplinking.md:7]: Add main use-cases here to help devs imagine when and why they could use deep-links. 

Suggestion:
- Create a link so your users can open your dapp directly in MetaMask app and interact with your dapp with their web3 account
- Provide a simple one-click experience such that users can easily make payments to another account (with pre-filled parameters like recipient address, amount, network, etc.)
- Does your dapp have a Connext payment channel open? Let your users make gasless and instant transactions with your dapp with Payment Channel requests! (note, this is subject to work only if the user has opted in for InstaPay experimental feature)

File: docs/guide/mobile-best-practices.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Mobile Best Practices
+
+If your site works with the MetaMask Extension, then you should be all set!
+If it doesn't, please refer to our [best practices](./mobile-best-practices.html) and if you're still having issues, please open an issue [here](https://github.com/MetaMask/metamask-mobile)
+
+# ethereum.initialized
```
Comment (omnat) [inline docs/guide/mobile-best-practices.md:6]: Is this section the provider section by @rekmarks ?

Note: best practices link is broken.

File: docs/.vuepress/config.js [hunk 3]
```diff
@@ -112,15 +112,18 @@ function getGuideSidebar(guide, api, bestPractices, mobile, resources) {
       title: mobile,
       collapsable: false,
       children: [
+        'mobile-getting-started',
+        'mobile-best-practices',
         'dapp-compatibility',
-
+        'deeplinking',
```
Comment (omnat) [inline docs/.vuepress/config.js:118]: Suggestion for structure:

1. Getting started
2. Make your site compatible with MetaMask  
3. Best practices 
4. Enhance your users' experience with your site/app 
(deeplinking, wallet connect)

File: docs/guide/walletconnect.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# WalletConnect
+
+You can use MetaMask mobile as a signer while using applications on another browser, desktop, or mobile application with WalletConnect.
+
+For a dapp developer, they wouldn’t need to do anything other than use our “official” qrcode-modal which already includes MetaMask deep linking
```
Comment (omnat) [inline docs/guide/walletconnect.md:5]: For a dapp developer, they wouldn’t need to do anything other than use **their** “official” qrcode-modal which already includes MetaMask deep linking

File: docs/guide/mobile-best-practices.md (hunk 2 above)
Comment (omnat) [reply docs/guide/mobile-best-practices.md:6]: Dapp compatibility to include MetaMask mobile as a listed wallet, and that link to open MetaMask app (if installed) or to go to play/app store (if not yet installed)

Simple guided-steps on letting your user use your apps on Metamask

Message to web3 app builders:
- send your users to Metamask app listing
- add Metamask mobile as option to connect wallet
- Support MM inpage provider and test so it works well

File: docs/guide/walletconnect.md (hunk 4 above)
Comment (jennypollack) [inline docs/guide/walletconnect.md:5]:
```suggestion
If your dapp supports Wallet Connect's official qrcode-modal, then your users can already use MetaMask Mobile for scanning the QR code and signing.
```

File: docs/guide/deeplinking.md (hunk 1 above)
Comment (jennypollack) [inline docs/guide/deeplinking.md:7]:
```suggestion
URLs embedded in QR-codes, hyperlinks in web-pages, emails or chat messages provide for robust cross-application signaling between very loosely coupled applications.

You can use deep-links for things like:

- Creating a link so your users can open your dapp directly in MetaMask app and interact with your dapp with their web3 account.

- Providing a one-click experience such that users can easily make payments to another account (with pre-filled parameters like recipient address, amount, network, etc.)

- Let your users make gasless and instant transactions with your dapp with Connext Payment Channel requests! (note, this is subject to work only if the user has opted in for InstaPay experimental feature)
```

**Pull request merged**